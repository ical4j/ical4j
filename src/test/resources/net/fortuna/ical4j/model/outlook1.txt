BEGIN:VCALENDAR
METHOD:REQUEST
VERSION:2.0
PRODID:-//Microsoft Corporation//Windows Live Calendar//EN
BEGIN:VTIMEZONE
TZID:Pacific Standard Time
BEGIN:STANDARD
DTSTART:20071104T020000
RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
TZOFFSETTO:-0800
TZOFFSETFROM:-0700
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20070311T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
TZOFFSETTO:-0700
TZOFFSETFROM:-0800
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
UID:9177c616-46dc-4755-bab1-5fdfecd31b6b
DTSTAMP:20140131T095408Z
CLASS:PUBLIC
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
TRANSP:OPAQUE
SEQUENCE:0
DTSTART;TZID=Pacific Standard Time:20140131T133000
DTEND;TZID=Pacific Standard Time:20140131T143000
SUMMARY:Event from outlook 2
DESCRIPTION:abc
PRIORITY:0
ATTENDEE;CUTYPE=INDIVIDUAL;CN=abc@web.de;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:abc@web.de
ORGANIZER;CN=Begenda T:MAILTO:abc@outlook.com
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-PT15M
END:VALARM
END:VEVENT
END:VCALENDAR