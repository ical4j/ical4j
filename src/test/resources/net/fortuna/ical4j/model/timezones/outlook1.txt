BEGIN:VCALENDAR
METHOD:REQUEST
VERSION:2.0
PRODID:-//Microsoft Corporation//Windows Live Calendar//EN
BEGIN:VTIMEZONE
TZID:Pacific Standard Time
BEGIN:STANDARD
DTSTART:20071104T020000
RRULE:FREQ=YEARLY;BYDAY=1SU;BYMONTH=11
TZOFFSETTO:-0800
TZOFFSETFROM:-0700
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20070311T020000
RRULE:FREQ=YEARLY;BYDAY=2SU;BYMONTH=3
TZOFFSETTO:-0700
TZOFFSETFROM:-0800
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
UID:0b12874e-0fb7-440a-a1cd-5fe2090c626a
DTSTAMP:20140221T110129Z
CLASS:PUBLIC
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
TRANSP:OPAQUE
SEQUENCE:1
DTSTART;TZID=Pacific Standard Time:20140221T133000
DTEND;TZID=Pacific Standard Time:20140221T143000
SUMMARY:From outlook edited
DESCRIPTION:dfgdg dfgfdgdfgd äöüß_<>\;„’’
LOCATION:room äöüß_<>\;„’’
PRIORITY:0
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@gmail.com;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@gmail.com
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@web.de;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@web.de
ORGANIZER;CN=Begenda T:MAILTO:begendatest01@outlook.com
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-PT15M
END:VALARM
END:VEVENT
END:VCALENDAR