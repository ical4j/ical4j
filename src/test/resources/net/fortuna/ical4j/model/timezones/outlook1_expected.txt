BEGIN:VCALENDAR
METHOD:REQUEST
VERSION:2.0
PRODID:-//Microsoft Corporation//Windows Live Calendar//EN
BEGIN:VTIMEZONE
TZID:America/Vancouver
BEGIN:STANDARD
DTSTART:20071104T020000
RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
TZOFFSETTO:-0800
TZOFFSETFROM:-0700
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20070311T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
TZOFFSETTO:-0700
TZOFFSETFROM:-0800
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
UID:0b12874e-0fb7-440a-a1cd-5fe2090c626a
DTSTAMP:20140221T110129Z
CLASS:PUBLIC
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
TRANSP:OPAQUE
SEQUENCE:1
DTSTART;TZID=America/Vancouver:20140221T133000
DTEND;TZID=America/Vancouver:20140221T143000
SUMMARY:From outlook edited
DESCRIPTION:dfgdg dfgfdgdfgd äöüß_<>\;„’’
LOCATION:room äöüß_<>\;„’’
PRIORITY:0
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@gmail.com;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@gmail.com
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@web.de;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@web.de
ORGANIZER;CN=Begenda T:MAILTO:begendatest01@outlook.com
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-PT15M
DESCRIPTION:display
END:VALARM
END:VEVENT
END:VCALENDAR